packages/components/src/styles/components/accordion.scss

```diff
@@ -116,5 +168,7 @@ $hds-accordion-item-border-radius: 6px;
 // CONTENT BLOCK
 
 .hds-accordion-item__content {
-  padding: 4px $hds-accordion-item-padding $hds-accordion-item-padding $hds-accordion-item-padding;
+  .hds-accordion-item & {
+    padding: 4px var(--hds-accordion-item-padding) var(--hds-accordion-item-padding) var(--hds-accordion-item-padding);
```

Comment:
@majedelass @andgen404 I think the `small` variant is set in Figma to 12px 4px 8px 8px. Can you double-check this, please?

Reply:
Addressed in 3e847d8